ui/index.d.ts

```diff
@@ -905,6 +905,11 @@ export interface IWizardOptions<T extends IActionContext> {
      * If true, step count will not be displayed for the entire wizard. Defaults to false.
      */
     hideStepCount?: boolean;
+
+    /**
+    * If true, a loading prompt will be displayed if there are long delays between wizard steps.
+    */
+    showLoadingPrompt?: boolean;
```

Comment:
One alternative is to only show the loading prompt if the delay is more than a certain interval (500ms? 1s?). It would prevent a flashing "Loading" in the vast majority of cases but still show it for most cases that need it

That being said, I'm fine with a boolean like this while we do more testing and see how it works in our various wizards

Reply:
I think I'll stay with the boolean for two reasons:

1. Flashing loading here won't 

> I just found a bug, and I think I'm going to need some help solving it. If you press escape during a loading prompt, then the loading prompt hides but the next step still appears.
> 
> Fixing it by throwing a `UserCancelledError` when in the `onDidHide` handler of the loading prompt won't work since when it I throw in the handler, catch/finally is not hit.
> 
> I'm thinking I'll have to use a Promise and reject it somewhere.

Talked with Eric offline, disabling cancelling during loading sounds like a good solution 😄